english/dream/dream.md

```diff
@@ -23,3 +23,5 @@ Afterwards, you decide to:
 [Drink a nice cup of tea](../tea/tea.md)
 
 [swim](../swim/swim.md)
+
+[Make a pancake](create-your-own-adventure/english/dream/pancakes/make-a-pancake.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.